designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleStateStatus.scala

```diff
@@ -3,15 +3,16 @@ package pl.touk.nussknacker.engine.api.deployment.simple
 import pl.touk.nussknacker.engine.api.deployment.ProcessActionType.ProcessActionType
```

Comment:
I wonder, maybe this file should be name Base/Dfault StateStatus?

Reply:
Good point, but for clarity I'd like to move it to another PR.